server/package.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "@aws-sdk/client-iot": "^3.78.0",
     "@aws-sdk/client-iot-wireless": "^3.50.0",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v8.1.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-reuse-twilio-numbers",
```

Comment:
Reminder to change this after https://github.com/bravetechnologycoop/brave-alert-lib/pull/48 is merged

Reply:
Done.